### DISCUSSION TITLE
Should `vue.inlayHints.missingProps` enabled by default?

### DISCUSSION THREAD

**Discussion by johnsoncodehk** [vuejs/language-tools#3416] [Polls]
`vue.inlayHints.missingProps` feature is disabled by default based on the principle of not disturbing users as much as possible. But with so many people thinking it should be enabled by default, I wonder if this feature deserves to be an exception.

It's not a majority vote, and we won't change it if there's more than 20% ~ 30% against it.

If you don't know what this feature is, see: https://twitter.com/Mokkapps/status/1678433961586970628

**Comment by rchl** [comment]
I've answered "no" because this idea of showing a missing prop as inlay hint seems like a bit of a hack. Missing prop is an error state so I wouldn't expect an inlay hint to represent that. Feels like misuse of inlay hint to me. Errors are expected to be represented by the usual red underline or whatever is the convention in a given editor. And this is already the case with Volar that missing properties trigger an error underline so I think that should be enough. And the functionality of adding the missing property should IMO be implemented as context-specific code action.

I'll hide this comment as to not potentially influence the answers but on the other hand, maybe those who said "yes" would also agree with me after reading my point.

**Comment by johnsoncodehk** [comment]
Sorry this may disappoint most people. Depending on the outcome of the vote, this feature will still disabled by default.
